src/util/modules.py

```diff
@@ -51,9 +51,28 @@
 from .records import ref, remove_menus, remove_records, remove_view, replace_record_references_batch
 
 INSTALLED_MODULE_STATES = ("installed", "to install", "to upgrade")
-NO_AUTOINSTALL = str2bool(os.getenv("UPG_NO_AUTOINSTALL", "0")) if version_gte("15.0") else False
 _logger = logging.getLogger(__name__)
 
+if version_gte("15.0"):
+    AUTO_INSTALL = os.getenv("UPG_AUTOINSTALL")
+    _NO_AUTOINSTALL = os.getenv("UPG_NO_AUTOINSTALL")
+    if AUTO_INSTALL:
+        if _NO_AUTOINSTALL:
```

Comment:
np
```suggestion
    if AUTO_INSTALL and _NO_AUTOINSTALL:
```

Reply:
No, this is not the same. The `else` part should then be changed to `elif not AUTO_INSTALL`.

I don't know if this make the code clearer or not...